handlers/clients.go

```diff
@@ -0,0 +1,26 @@
+package handlers
+
+import (
+	"context"
+	"net/url"
+
+	searchC "github.com/ONSdigital/dp-api-clients-go/site-search"
+)
+
+//go:generate moq -out ../mocks/clients.go -pkg mocks . RenderClient SearchClient
+
+// ClientError is an interface that can be used to retrieve the status code if a client has errored
+type ClientError interface {
+	Error() string
+	Code() int
```

Comment:
I understand this may be copied from somewhere else but I really think StatusCode() is more appropriate than Code() for retrieving the status code.. It falls in line with implementations found elsewhere in the Go world and is more descriptive.. 'Code' could mean 'Exit Code' for example.

Reply:
Furthermore - I can't see where this is being used? But I do remember seeing this implementation somewhere else (where it is used to assert an error check it's statusCode) and remember noticing that it was actually unnecessary to have the full error as an interface - the interface just needs to be for Code() (or StatusCode() :p) function to ascertain the statusCode. With interfaces it's better to include only the behaviour you need.

Reply:
Yeah I agree. This is a very good point. It should be specific. I can't change the name to StatusCode() as I would need to change it in the `dp-api-clients-go` which means refactoring all the apps that use the renderer client for example. This would require a lot of changes throughout apps

Reply:
I just wanted to add to what Justin has said and it would make more sense to update the library `dp-api-clients-go` before rolling out a change to any apps regarding retrieval of status code; but agree `StatusCode()` would be better.